web-registry/src/pages/MyEcocredits/hooks/useUpdateTxModalTitle.tsx

```diff
@@ -0,0 +1,38 @@
+import { DeliverTxResponse } from '@cosmjs/stargate';
+import { useEffect } from 'react';
+import { useStateSetter } from '../../../types/react/use-state';
+
+type Props = {
+  deliverTxResponse?: DeliverTxResponse;
+  setTxModalTitle: useStateSetter<string | undefined>;
+};
+
+const SELL_ORDER_EVENT_TYPE = 'regen.ecocredit.marketplace.v1.EventSell';
+const SELL_ORDER_ATTRIBUTE_KEY = 'sell_order_id';
+
+export const useUpdateTxModalTitle = ({
```

Comment:
This hook is used to update TxModalTitle based on information in the transaction response.
For the sell order we want to display `Sell Order #xxx`